src/locales/kafka-ui.json

```
@@ -287,6 +287,14 @@
     "reset_offset_connected_alert_body": "One or more members of the consumer group are currently connected to a topic. All members of a consumer group must be disconnected before resetting the offset.",
     "offset_successfully_reset": "offsets for the consumer group have been successfully reset",
     "delete_consumer_connected_alert_title": "The {{name}} consumer group cannot be deleted at this time",
-    "delete_consumer_connected_alert_body": "One or more members are currently connected to a topic. All members of a consumer group that are connected to topic must be disconnected before deleting the consumer group."
+    "delete_consumer_connected_alert_body": "One or more members are currently connected to a topic. All members of a consumer group that are connected to topic must be disconnected before deleting the consumer group.",
+    "state": {
+      "stable_state": "Stable",
+      "empty_state": "Empty",
+      "dead_state": "Dead",
+      "completing_rebalance_state": "CompletingRebalance",
+      "preparing_rebalance_state": "PreparingRebalance",
+      "unknown_state": "Unknown"
```

```suggestion
      "stable": "Stable",
      "empty": "Empty",
      "dead": "Dead",
      "completing_rebalance": "CompletingRebalance",
      "preparing_rebalance": "PreparingRebalance",
      "unknown": "Unknown"
```

Nitpicking, but since `state` is already part of the key because of the parent, no need to have it again.